addon/components/columns/base.hbs

```diff
@@ -26,6 +26,6 @@
     @table={{this.table}}
     @resizeOnDrag={{this.resizeOnDrag}}
     @isResizing={{mut this.isResizing}}
-    @onColumnResized={{action this.onColumnResized this.column}}
+    @onColumnResized={{fn this.columnResized this.column}}
```

Comment:
Can we put these all back to their old names? This should be `onColumnResized`

Reply:
The names were changed because it generates an infinite loop error. Ember  recursively calls the action because it can't distinguish between the function and passed parameter.

```js
 @action
 onColumnResized(/* column, width */) {
   this.onColumnResized?.(...arguments);
 }
```

My fix was to rename the internal functions and keep the parameter names the same.

```js
  @action
  thOnColumnResized(/* column, width */) {
    this.onColumnResized?.(...arguments);
  }
```

This can be fixed when converted to  glimmer: `this.args.onColumnResized?.()`. In the meantime I can change the names to something else and/or put TODO comments in the code to rename them when converted to glimmer.

Reply:
There should be no wrapping function. Wherever there was an `action` that just called a passed function, we can just remove the action entirely.

Reply:
Dug into the wrapping functions again.  The functions are passing optional actions to other components.  

In most cases, the target component uses the `{{fn}}` helper with the passed function.  So a valid function is required.

I found a related SO [question](https://stackoverflow.com/questions/58847532/how-do-i-make-an-optional-action-for-an-octane-component) for actions but didn't think these were good solutions in this case.  

I passed the functions directly  were I could and cleaned up the remaining wrapped function names.  While the names don't match the original names they're better than what I used before.

While there may be a better solution here I did not want to drag out this PR, if it could be solved later